linux/check_os_fullname/miracle_fullname_map.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Guest id "asianux8_64Guest" and full name "MIRACLE LINUX 8 (64-bit)" is available on ESXi 6.7P06 or later
+
+- name: "Set fact of expected guest id on ESXi {{ esxi_version }}"
+  ansible.builtin.set_fact:
+    expected_guest_major_ver: "{{ [guest_os_ansible_distribution_major_ver | int, 8 ] | min }}"
```

Comment:
This task can also be put under below block as this variable is only used in the block for Miracle Linux.